app/services/doc_auth/lexis_nexis/doc_pii_reader.rb

```diff
@@ -0,0 +1,87 @@
+module DocAuth
+  module LexisNexis
+    module DocPiiReader
+      PII_EXCLUDES = %w[
+        Age
+        DocSize
+        DOB_Day
+        DOB_Month
+        DOB_Year
+        ExpirationDate_Day
+        ExpirationDate_Month
+        ExpirationDate_Year
+        FullName
+        Portrait
+        Sex
+      ].freeze
+
+      private
```

Comment:
Similar to the question I asked in ImageMetricsReader: What's the benefit of calling `private` here?  (Not saying there isn't a benefit, just trying to understand. Some of this wasn't private in TrueIDResponse.)

Reply:
@eileen-nava , meaning this is intended to be used in including class internally?

Reply:
Got it, thanks. 🙏🏻